.dockerignore

```diff
@@ -1,2 +1,6 @@
+.dockerignore
```

Comment:
That's super meta 😸

Reply:
😂 yes

Reply:
I usually add the Dockerfile itself as well, didn't take time to look into which Dockerfile may be used docker-in-docker